circuits/ts/__tests__/Hasher.test.ts

```diff
@@ -19,29 +18,44 @@ import {
     hash4,
     hash3,
 } from 'maci-crypto'
+import * as path from 'path'
+import { expect } from 'chai'
+
+describe('Poseidon hash circuits', function() {
+    this.timeout(30000)
```

Comment:
Why is this necessary?

Reply:
testing frameworks have some default timeout (you can usually set it globally or per test file) - given the nature of these tests, they can take a long time to run and default timeouts might be triggered